Cargo.toml

```diff
@@ -13,13 +13,24 @@ keywords = ["integer", "u256", "ethereum"]
 categories = ["cryptography::cryptocurrencies", "mathematics", "no-std"]
 
 [workspace]
-members = [
-  "bench",
-  "intrinsics",
-]
+members = ["bench", "intrinsics"]
 
 [features]
 llvm-intrinsics = ["ethnum-intrinsics"]
+serde = ["impl-serde", "crate-serde"]
+scale = ["parity-scale-codec", "derive_more"]
+default = ["serde"]
```

Comment:
I don't think `serde` support should be built by default.

Reply:
Done